docs/en/sql-reference/ansi.md

```diff
@@ -25,14 +25,14 @@ The following table lists cases when query feature works in ClickHouse, but beha
 |------------|--------------------------------------------------------------------------------------------------------------------------|----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | **E011**   | **Numeric data types**                                                                                                   | **Partial**{.text-warning} |                                                                                                                                                                                           |
 | E011-01    | INTEGER and SMALLINT data types                                                                                          | Yes {.text-success}         |                                                                                                                                                                                           |
-| E011-02    | REAL, DOUBLE PRECISION and FLOAT data types data types                                                                   | Partial {.text-warning}     | `FLOAT(<binary_precision>)`, `REAL` and `DOUBLE PRECISION` are not supported                                                                                                              |
+| E011-02    | REAL, DOUBLE PRECISION and FLOAT data types data types                                                                   | Partial {.text-warning}     | `FLOAT(<binary_precision>)` and `REAL` are not supported                                                                                                                                  |
```

Comment:
`REAL` is supported too: https://github.com/ClickHouse/ClickHouse/blob/89c64cbdfc1db9899268780bbe03363770c98fa1/src/DataTypes/DataTypesNumber.cpp#L66

`FLOAT(<binary_precision>)` also should be supported, see #18696